### PR TITLE
Update forms, add concepts plus translations and new dispositions for…

### PR DESCRIPTION
**Pull request by MiguelAHPpih**
… COR

### CODE REVIEW

File: configuration/pih/htmlforms/vitals.xml [hunk 1]
```diff
@@ -0,0 +1,659 @@
...
+  ~
+  ~ Software distributed under the License is distributed on an "AS IS"
+  ~ basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing rights and limitations
+  ~ under the License.
+  ~
+  ~ Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+  -->
+
+<htmlform id="vitals-form" formUuid="68728aa6-4985-11e2-8815-657001b58a90"
```
Comment (brandones) [inline configuration/pih/htmlforms/vitals.xml:15]: Could you comment any differences between this and the default Vitals form that it's overriding?

File: configuration/pih/htmlforms/vitals.xml [hunk 2]
```diff
@@ -0,0 +1,659 @@
...
+            </fieldset>
+
+        </includeIf>
+
+        <fieldset field-separator=" ">
+            <legend><uimessage code="mirebalais.vitals.temperature.title"/></legend>
+            <h3><uimessage code="mirebalais.vitals.temperature.title"/></h3>
+
+            <p class="left">
+                <obs conceptId="PIH:TEMPERATURE (C)" id="temperature_c" showUnits="emr.units.degreesCelsius" unitsCssClass="append-to-value"/>
```
Comment (MiguelAHPpih) [inline configuration/pih/htmlforms/vitals.xml:535]: Remove Farenheit mesure

File: configuration/pih/htmlforms/vitals.xml [hunk 3]
```diff
@@ -0,0 +1,659 @@
...
+            </p>
+
+        </fieldset>
+
+        <fieldset field-separator=" ">
+            <legend><uimessage code="mirebalais.vitals.weight.title"/></legend>
+            <h3><uimessage code="mirebalais.vitals.weight.title"/></h3>
+
+            <p class="left">
+                <obs conceptId="PIH:WEIGHT (KG)" id="weight_kg" showUnits="emr.units.kilograms" unitsCssClass="append-to-value"/>
```
Comment (MiguelAHPpih) [inline configuration/pih/htmlforms/vitals.xml:500]: Removed pounds measure

File: configuration/pih/htmlforms/vitals.xml [hunk 4]
```diff
@@ -0,0 +1,659 @@
...
+                </fieldset>
+            </includeIf>
+        </ifMode>
+
+        <fieldset field-separator=" ">
+            <legend><uimessage code="mirebalais.vitals.height.title"/></legend>
+            <h3><uimessage code="mirebalais.vitals.height.title"/></h3>
+
+            <p class="left">
+                <obs conceptId="PIH:HEIGHT (CM)" id="height_cm" showUnits="emr.units.centimeters" unitsCssClass="append-to-value"/>
```
Comment (MiguelAHPpih) [inline configuration/pih/htmlforms/vitals.xml:490]: Removed meters measure

**Review by brandones — APPROVED** [review]
Awesome, thanks @MiguelAHPpih !

**Pull request merged**